inst/include/state_saver.hpp

```diff
@@ -158,11 +186,20 @@ class ChildModelStateSaver<ChildModel, real_type> {
     children_state.hc1_hiv_pop.chip(i, children_state.hc1_hiv_pop.NumDimensions - 1) =
         state.children.hc1_hiv_pop;
     children_state.hc2_hiv_pop.chip(i, children_state.hc2_hiv_pop.NumDimensions - 1) =
-      state.children.hc2_hiv_pop;
+        state.children.hc2_hiv_pop;
     children_state.hc1_art_pop.chip(i, children_state.hc1_art_pop.NumDimensions - 1) =
-      state.children.hc1_art_pop;
+        state.children.hc1_art_pop;
     children_state.hc2_art_pop.chip(i, children_state.hc2_art_pop.NumDimensions - 1) =
-      state.children.hc2_art_pop;
+        state.children.hc2_art_pop;
+    children_state.hc1_noart_aids_deaths.chip(i, children_state.hc1_noart_aids_deaths.NumDimensions - 1) =
+      state.children.hc1_noart_aids_deaths;
+    children_state.hc2_noart_aids_deaths.chip(i, children_state.hc2_noart_aids_deaths.NumDimensions - 1) =
+      state.children.hc2_noart_aids_deaths;
+    children_state.hc1_art_aids_deaths.chip(i, children_state.hc1_art_aids_deaths.NumDimensions - 1) =
+      state.children.hc1_art_aids_deaths;
+    children_state.hc2_art_aids_deaths.chip(i, children_state.hc2_art_aids_deaths.NumDimensions - 1) =
+      state.children.hc2_art_aids_deaths;
+    //TO DO: ask rob how to save out an integer (hc_art_num)
```

Comment:
Should be able to do this by just assigning a value at index `i`

```
children_state.hc_art_num(i) = state.children.hc_art_num;
```

Reply:
Actually have decided it doesn't need to be output! only in the statespace because we need t-1 and t values for it so can't be in intermediate